dali/base/dafdesc.cpp

```diff
@@ -3701,9 +3701,14 @@ class CStoragePlaneInfo : public CInterfaceOf<IStoragePlane>
     {
         return xml->queryProp("storageapi/@account");
     }
-    virtual const char * queryStorageContainer() const override
+    virtual StringBuffer & getStorageContainer(StringBuffer & container, unsigned stripeNumber) const override
     {
-        return xml->queryProp("storageapi/@container");
+        StringBuffer path("storageapi/");
+        if (stripeNumber==0)
+            path.append("@container");
+        else
+            path.appendf("containers/@stripe%d", stripeNumber);
```

Comment:
I was expecting containers[%d]/ so that the helm would be

```
containers: [ a, b, c, d ]
```
or
```
containers:
- a
- b
- c
```

```
containers:
  stripe1: a
  stripe2: b
  stripe3: c
```

doesn't seem so natural.  @jakesmith thoughts?

Reply:
@jakesmith @ghalliday What form should I implement?  The first form seems cleaner/neater.  The second form is self-documenting - easier to understand.

Reply:
I agree, when looking at the code above, I was expecting it would be dealing with a list of containers.